pkg/security/activitydump/activity_dump.go

```diff
@@ -995,6 +995,10 @@ func (ad *ActivityDump) Unzip(inputFile string, ext string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("couldn't unzip %s: %w", inputFile, err)
 	}
+	err = outputFile.Close()
+	if err != nil {
+		return "", fmt.Errorf("could not close close file [%s]: %w", outputFile.Name(), err)
```

Comment:
typo (2 close)

Reply:
It's intentional. It is safe to call `Close()` twice. I kept the `defer outputFile.Close()` to handle the case where `io.Copy(outputFile, gzipReader)` returns an error. Another way is to close the file if the copy fails. In this case, if we do not want to lose the error, we can use `errors.NewAggregate()` as I did in my last commit.

Reply:
i meant in the fmt.Errorf text